### ISSUE TITLE
Issue with non-ascii characters at of line?

### ISSUE THREAD

**Issue by Sarcasm** [janestreet/patdiff#2]
Hello,

Sorry about the "mysterious" title, I'm not sure to understand the issue.

I was able to reduce the issue to the following test case:

```diff
$ cat old.el
(message "hello")
$ cat new.el
(message "bonjour")
$ patdiff old.el new.el 
------ old.el
++++++ new.el
@|-1,1 +1,1 ============================================================
-|(message "hello
+|(message "bonjour
```

The issue is the `")` discarded at the end, the expected output is:

```diff
------ old.el
++++++ new.el
@|-1,1 +1,1 ============================================================
-|(message "hello")
+|(message "bonjour")
```

p.s: thanks for patdiff, it's a quite nice diff program.

**Comment by bmillwood** [comment]
I can reproduce this issue with the latest external release of patdiff but not with the latest internal version, so I'm hopeful it will be resolved by the next release. Let's check again then.

**Comment by zph** [comment]
@bmillwood This is fixed in `113.00.00`

Thanks for patdiff.

Log showing desired behavior & version number:

```
λ tmp~ opam info patdiff
             package: patdiff
             ...elided...
   installed-version: 113.00.00 [system]

λ tmp~ cat old.el new.el
(message 'hello')
(message 'bonjour')
λ tmp~ patdiff old.el new.el
------ old.el
++++++ new.el
@|-1,1 +1,1 ============================================================
-|(message 'hello')
+|(message 'bonjour')
```

**Comment by bmillwood** [comment]
Thanks for the confirmation.
